### PR TITLE
Additional text changes

### PR DESCRIPTION
**Pull request by osscca**

### CODE REVIEW

File: website/ninja_web/templates/user_detail.html [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
       {{request.user}}
       {% if user_page == request.user %}
-      <h4>Welcome {{ user.name }} to your dojo. Put your shoes at the door and come in.</h4>
+      <h4>Welcome {{ user.name }} to your dojo. Leave your shoes at the door and come in.</h4>
       {% else %}
       <h4>Qualified ninja since {{ user_page.date_joined|date:"F Y" }}.</h4>
       {% endif %}
 
-      <h2>Submitted {{ plugins.count }} plugin{{ plugins.count|pluralize }} until now.</h2>
+      <h2>Submitted {{ plugins.count }} plugins{{ plugins.count|pluralize }} until now.</h2>
```
Comment (pmourelle) [inline website/ninja_web/templates/user_detail.html]: In this case is correct the original. The "pluralize" filter adds and 's' if needed.

**Comment by pmourelle** [comment]
Again: great corrections!.
But as we always say: a great correction comes with a great counter-correction :P
Can you change the commented line? Then I'll merge it. :)
Thanks again!

**Comment by osscca** [comment]
Ah, and oops again... I should have known.

**Comment by osscca** [comment]
Fixed it :)

**Comment by pmourelle** [comment]
Done. Thanks again! :)

**Pull request merged**